mac.sh

```diff
@@ -192,7 +192,7 @@ install_oh_my_zsh() {
 }
 
 install_elasticsearch() {
-  cask_install_or_upgrade 'homebrew/cask-versions/java8'
+  cask_install_or_upgrade 'homebrew/cask-versions/adoptopenjdk8'
```

Comment:
Is this a High Sierra requirement? Just curious

Reply:
No, the java bundle that the java8 cask points to is actually no longer made available.

Reply:
lol even better

Reply:
And they thought nobody needed it...